src/lib/addClipMarkup.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Add clip-related markup to page head for Google SEO.
+ *
+ * @param {player} Player An instance of the Player.
+ * @return {void}
+ */
+// TODO: NEED TO DETERMINE WHERE THIS METHOD SHOULD BE DEFINED, IF NOT HERE
+export function addClipMarkup(player) {
```

Comment:
I'm not sure how much it is followed anymore, but in general I think it is best to leave the term “clip” out of public code.

Reply:
Updated 👍